### ISSUE TITLE
gluestick training time

### ISSUE THREAD

**Issue by fatih4d** [cvg/glue-factory#81]
I am trying to train gluestick with deep lsd config. I am using 2 A10 GPUs for training. The training has been going on for 5 days and the loss is at around 2.3 . In one of the repo issues, there was information about 2 days for the training period. Do you have any comments about it taking so long on my experiment?

**Comment by rpautrat** [comment]
Hi, I am not sure where you the 2 days training, but without pre-training, I would expect the training to last around 7 days on homography pre-training, and another 7 days for the fine-tuning on MegaDepth. This can be made faster with more GPUs of course.
We are also working on a lighter version that takes less time to train, and will release it once it is ready.

**Comment by fatih4d** [comment]
this is where I saw the information related with the training times https://github.com/cvg/glue-factory/issues/69#issuecomment-2043167473 , thanks for the detailed info

**Comment by rpautrat** [comment]
Ah, this was the timing for LightGlue, not for GlueStick :)
